deepmd/pt_expt/utils/comm.py

```
@@ -50,30 +50,36 @@ def _check_underlying_ops_loaded() -> None:
-        except Exception:
-            # If deepmd.pt itself fails to import, fall through to the
-            # explicit RuntimeError below — clearer than re-raising a
-            # potentially-unrelated import error.
-            pass
-
-    if not (
-        hasattr(torch.ops, "deepmd_export")
-        and hasattr(torch.ops.deepmd_export, "border_op")
-        and hasattr(torch.ops.deepmd_export, "border_op_backward")
-    ):
+        except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)